analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/metrics/CalendarWeek.kt

```diff
@@ -19,7 +20,15 @@ internal data class CalendarWeek(private val week: Int, private val year: Int) :
         }
 
         fun numberOfWeeksBetween(a: CalendarWeek, b: CalendarWeek): Int {
-            return b.week + 52 * b.year - (a.week + 52 * a.year)
+            return ChronoUnit.WEEKS.between(getWeekDate(a.year, a.week), getWeekDate(b.year, b.week)).toInt()
+        }
+
+        private fun getWeekDate(year: Int, week: Int): OffsetDateTime? {
+            return OffsetDateTime.now().withYear(year)
+                .with(WeekFields.ISO.weekOfWeekBasedYear(), week.toLong())
+                .with(WeekFields.ISO.dayOfWeek(), 1)
+                .withHour(12).withMinute(0).withSecond(0).withNano(0)
+                .withOffsetSameInstant(ZoneOffset.UTC)
```

Comment:
This is not immediately obvious what this does. I suggest to add a short comment what this tries to solve :-)